pfcli/service/__init__.py

```diff
@@ -32,6 +32,7 @@ class ServiceType(str, Enum):
     GROUP_PROJECT_CHECKPOINT = "GROUP_PROJECT_CHECKPOINT"
     JOB_WS = "JOB_WS"
     DEPLOYMENT = "DEPLOYMENT"
+    DEPLOYMENT_SCALE = "DEPLOYMENT_SCALE"
     DEPLOYMENT_LOG = "DEPLOYMENT_LOG"
     DEPLOYMENT_METRICS = "DEPLOYMENT_METRICS"
     PFS_PROJECT_USAGE = "PFS_PROJECT_USAGE"
```

Comment:
So, we do not have to change any of these commands, right?

Reply:
I already checked `usage` command, and it showed sum of all replicas' running time.
I'll check `metric` command! Thank you for the reminder.